### PR TITLE
Add Arrland adapter

### PR DESCRIPTION
**Pull request by arrland**
Name: Pirates of the Arrland
Twitter Link: https://x.com/ArrlandNFT
List of audit links if any: https://github.com/solidproof/projects/tree/main/2024/Arrland
https://skynet.certik.com/projects/arrland 
https://github.com/solidproof/projects/tree/main/2024/Pirates%20of%20the%20Arrland
Website Link: https://arrland.com/
Logo (High resolution, will be shown with rounded borders): https://imgur.com/WelrkvN.png
Current TVL:

--- polygon-staking ---
RUM                       928.9489624865132
Total: 928.9489624865132 

--- polygon-pool2 ---
RUM                       11.11 k
WMATIC                    9.15 k
USDT                      1.97 k
Total: 22.23 k 

--- polygon ---
USD+                      51.66 k
Total: 51.66 k 

--- staking ---
RUM                       929
Total: 928.9489624865132 

--- pool2 ---
RUM                       11.11 k
WMATIC                    9.15 k
USDT                      1.97 k
Total: 22.23 k 

--- tvl ---
USD+                      51.66 k
Total: 51.66 k 

------ TVL ------
polygon                   51.66 k
polygon-pool2             22.23 k
pool2                     22.23 k
polygon-staking           929
staking                   929

total                    51.66 k

Treasury Addresses (if the protocol has treasury) - None
Chain: Polygon
Coingecko ID (so your TVL can appear on Coingecko, leave empty if not listed): "arrland-rum"
Coinmarketcap ID (so your TVL can appear on Coinmarketcap, leave empty if not listed): 24789
Short Description (to be shown on DefiLlama): Pirates of the Arrland is a Play2Earn competetive 3D game project combining DeFi and on-chain farming.
Token address and ticker if any: 0x14E5386f47466A463f85D151653E1736c0c50Fc3
Category (full list at https://defillama.com/categories) *Please choose only one: Gaming
forkedFrom (Does your project originate from another project): No
methodology: Counts total value locked on Pirate NFTs staked, the token staking, LP token staking
Github org/user (Optional, if your code is open source, we can track activity): https://github.com/arrland

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/arrland exports TVL: 
        
 
 
    polygon                   51.92 k
    polygon-pool2             22.23 k
    pool2                     22.23 k
    polygon-staking           929
    staking                   929
    
    total                    51.92 k

File: projects/arrland/index.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  const maticPrice = await getMaticPrice(api);
+  const floorPriceUSD = floorPrice * maticPrice;
+
+  const tvl_sum = totalNFTs * floorPriceUSD;
+  return { 'usd': tvl_sum };
+}
+
+module.exports = {
+  polygon: {
+    tvl,
```
Comment (g1nt0ki) [inline projects/arrland/index.js:77]: sorry, we cant count staked NFT as tvl, other projects have requested the same and we said no to them as well. Atm, the only NFTs we price are those with chainlink oracle support on Ethereum

**Comment by llamatester** [comment]
The adapter at projects/arrland exports TVL: 
        
 
 
    polygon-pool2             36.02 k
    pool2                     36.02 k
    polygon                   21.95 k
    polygon-staking           2.25 k
    staking                   2.25 k
    
    total                    21.95 k

File: projects/arrland/index.js (hunk 1 above)
Comment (arrland) [reply projects/arrland/index.js]: I pushed new update

Update TVL calculation for Arrland on Polygon

- Remove custom TVL calculation based on NFT floor price
- Simplify TVL calculation using sumTokensExport helper
- TVL now counts $RUM tokens locked in LP staking contracts
- Update methodology description to reflect new calculation method

Comment (g1nt0ki) [reply projects/arrland/index.js]: hi @arrland RUM locked in LP staking contracts are reward of LP providers right? why is it marked as tvl?

Comment (arrland) [reply projects/arrland/index.js]: @g1nt0ki reward for LP staking is not tvl ?

Ok will remove it.

**Comment by llamatester** [comment]
The adapter at projects/arrland exports TVL: 
        
 
 
    polygon-pool2             40.27 k
    pool2                     40.27 k
    polygon-staking           3.78 k
    staking                   3.78 k
    polygon                   0
    
    total                    0

File: projects/arrland/index.js (hunk 1 above)
Comment (arrland) [reply projects/arrland/index.js]: @g1nt0ki now tvl is 0

**Comment by g1nt0ki** [comment]
@arrland thanks for fixing it

**Pull request merged**